tests/integration/tests/cluster/test-cluster-role-bindings.spec.js

```diff
@@ -96,13 +92,15 @@ context('Test Cluster Role Bindings', () => {
       .type('test-group')
       .blur({ force: true });
 
-    cy.get('ui5-dialog')
-      .contains('ui5-button', 'Update')
-      .should('be.visible')
+    cy.get('.edit-form')
+      .find('.header-actions')
+      .contains('ui5-button:visible', 'Save')
       .click();
```

Comment:
I added a specific command for this action, so you can just write:
`cy.saveChanges('Edit');`